src/node/util.ts

```diff
@@ -509,3 +509,17 @@ export const isFile = async (path: string): Promise<boolean> => {
     return false
   }
 }
+
+/**
+ * Escapes any HTML string special characters, like &, <, >, ", and '.
+ *
+ * Source: https://stackoverflow.com/a/6234804/3015595
+ **/
+export function escapeHtml(unsafe: string): string {
+  return unsafe
+    .replace(/&/g, "&amp;")
```

Comment:
seems kinda unfortunate that there isn't a built-in function for doing this? I'm always wary of escaping things incorrectly, but this seems reasonable enough

Reply:
Well, it is [built-in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape) but:
> Although `escape()` is not strictly deprecated (as in "removed from the Web standards"), it is defined in Annex B of the ECMA-262 standard, whose introduction states:
> Programmers should not use or assume the existence of these features and behaviors when writing new ECMAScript code